File: GOVI-Back/src/main/java/com/sncf/govi/service/model/Conducteur.java
package com.sncf.govi.service.model;

import lombok.Builder;
import lombok.Data;

@Builder
@Data
public class Conducteur {
    private String residence;
    private String codeADC;
    private Boolean isPS;
    private Boolean isFS;
    @Builder.Default
    private String couleur = CouleurEnum.ROUGE.getCode();
}

File: GOVI-Front/src/app/model/Conducteur.ts
import { Releve } from './Releve';

export class Conducteur {
  codeADC: string;
  residence: string;
  isPS: boolean;
  isFS: boolean;
  couleur: string;
  releves: Releve[];

  constructor(
    codeADC: string,
    residence: string,
    isPS: boolean,
    isFS: boolean,
    couleur: string,
    releves: Releve[]
  ) {
    this.couleur = couleur;
    this.isFS = isFS;
    this.isPS = isPS;
    this.codeADC = codeADC;
    this.residence = residence;
    this.releves = releves;
  }
}
